### PR TITLE
Presets: support Marked description layout

### PR DESCRIPTION
**Pull request by limonspb**
Allowing [Marked ](https://marked.js.org/)for presets description to make them nicer.

new optional metaproperty:
`#$ PARSER: MARKED/TEXT`

default is TEXT

Corresponding presets PR:
https://github.com/betaflight/firmware-presets/pull/193

Example:

image

Dompurify library is added for safety to sanitize html.

To test:
Download Configurator from this PR.
Add custom preset source - yours or this one:
link: https://github.com/limonspb/firmware-presets
branch: marked-demo

### CODE REVIEW

File: src/tabs/presets/DetailedDialog/PresetsDetailedDialog.js
Comment (chmelevskij) [inline src/tabs/presets/DetailedDialog/PresetsDetailedDialog.js:87]: Not really sure why need dom purify here? `marked` should already be sanitized

Comment (limonspb) [reply src/tabs/presets/DetailedDialog/PresetsDetailedDialog.js:87]: i was surprized too, but they say it is not sanitizing and recommending DOMPurify
https://marked.js.org/

Comment (chmelevskij) [reply src/tabs/presets/DetailedDialog/PresetsDetailedDialog.js:87]: Ah, weird but ok 😄

**Review by chmelevskij — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
